test/integration/work_test.go

```diff
@@ -554,4 +554,218 @@ var _ = ginkgo.Describe("ManifestWork", func() {
 			util.AssertAppliedManifestWorkDeleted(appliedManifestWork.Name, spokeWorkClient, eventuallyTimeout, eventuallyInterval)
 		})
 	})
+
+	ginkgo.Context("Resource sharing and adoption between manifestworks", func() {
```

Comment:
an adoption workflow requires something  like
1. create configmap
2. create manifestwork with same configmap content
3. ensure the manifestwork does not delete the configmap and recreate it. The existing configmap should be preserved.